utilities/backup/backup_engine_test.cc

```diff
@@ -1101,6 +1101,148 @@ class BackupEngineTestWithParam : public BackupEngineTest,
   }
 };
 
+TEST_F(BackupEngineTest, IncrementalRestore) {
+  const int keys_iteration = 100;
+
+  options_.disable_auto_compactions = true;
+  options_.level0_file_num_compaction_trigger = 1000;
+
+  std::vector<std::string> always_copyable_files = {
```

Comment:
Explicit list of files like this is fragile to changes in DB operations that might change the order or when we allocate new file numbers. You might have taken inspiration from existing testings in this same test file, but those are subtly either (a) constructing a dummy DB from a set of file names and looking at how they are handled, or (b) injecting extra files into the backup dir to be cleaned up. Do you think we can avoid this?